packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -621,17 +624,20 @@ export class AnchoredRegion extends FASTElement {
         this.pendingPositioningUpdate = false;
 
         let regionRect: DOMRect | ClientRect | null = null;
+        regionRect = this.applyIntersectionEntries(entries);
```

Comment:
what's the purpose of assigning this to `null` and then to the product of `this.applyIntersectionEntries(entries)`?

Reply:
cleaned up